packages/botonic-react/src/util/react.js

```diff
@@ -18,3 +19,14 @@ export const mapObjectNonBooleanValues = obj => {
     return [key, value]
   })
 }
+
+const hasChildren = element =>
+  isValidElement(element) && Boolean(element.props.children)
+
+export const hasComplexChildren = element =>
+  isValidElement(element) &&
+  hasChildren(element) &&
```

Comment:
In `hasChildren` you're calling again the `isValidElement` of `element`, I think the `isValidElement` from line 27 can be removed.

Reply:
good catch! changed 👍